### ISSUE TITLE
mpl style not found

### ISSUE THREAD

**Issue by tbennett4** [rsagroup/rsatoolbox#196]
```
---------------------------------------------------------------------------
FileNotFoundError                         Traceback (most recent call last)
~\Documents\Anaconda_Python\lib\site-packages\matplotlib\style\core.py in use(style)
    120             try:
--> 121                 rc = rc_params_from_file(style, use_default_template=False)
    122                 _apply_style(rc)

~\Documents\Anaconda_Python\lib\site-packages\matplotlib\__init__.py in rc_params_from_file(fname, fail_on_error, use_default_template)
    873     """
--> 874     config_from_file = _rc_params_in_file(fname, fail_on_error=fail_on_error)
    875 

~\Documents\Anaconda_Python\lib\site-packages\matplotlib\__init__.py in _rc_params_in_file(fname, transform, fail_on_error)
    802     rc_temp = {}
--> 803     with _open_file_or_url(fname) as fd:
    804         try:

~\Documents\Anaconda_Python\lib\contextlib.py in __enter__(self)
    112         try:
--> 113             return next(self.gen)
    114         except StopIteration:

~\Documents\Anaconda_Python\lib\site-packages\matplotlib\__init__.py in _open_file_or_url(fname)
    780             encoding = "utf-8"
--> 781         with open(fname, encoding=encoding) as f:
    782             yield f

FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\tcbht\\Documents\\Anaconda_Python\\lib\\site-packages\\rsatoolbox\\vis\\rdm.mplstyle'

The above exception was the direct cause of the following exception:

OSError                                   Traceback (most recent call last)
<ipython-input-78-1b0d3749e5d9> in <module>
----> 1 vis.show_rdm(eprdm)

~\Documents\Anaconda_Python\lib\site-packages\rsatoolbox\vis\rdm_plot.py in show_rdm(rdm, pattern_descriptor, cmap, rdm_descriptor, n_column, n_row, show_colorbar, gridlines, num_pattern_groups, figsize, nanmask, style, vmin, vmax, icon_spacing, linewidth)
    138     # some are global for figure, others local. Perhaps dicts indexed by axis is easiest
    139     return_handles = collections.defaultdict(dict)
--> 140     with plt.style.context(style):
    141         fig, ax_array = plt.subplots(
    142             nrows=int(n_row),

~\Documents\Anaconda_Python\lib\contextlib.py in __enter__(self)
    111         del self.args, self.kwds, self.func
    112         try:
--> 113             return next(self.gen)
    114         except StopIteration:
    115             raise RuntimeError("generator didn't yield") from None

~\Documents\Anaconda_Python\lib\site-packages\matplotlib\style\core.py in context(style, after_reset)
    158         if after_reset:
    159             mpl.rcdefaults()
--> 160         use(style)
    161         yield
    162 

~\Documents\Anaconda_Python\lib\site-packages\matplotlib\style\core.py in use(style)
    122                 _apply_style(rc)
    123             except IOError as err:
--> 124                 raise IOError(
    125                     "{!r} not found in the style library and input is not a "
    126                     "valid URL or path; see `style.available` for list of "

OSError: 'C:\\Users\\tcbht\\Documents\\Anaconda_Python\\lib\\site-packages\\rsatoolbox\\vis\\rdm.mplstyle' not found in the style library and input is not a valid URL or path; see `style.available` for list of available styles
```

**Comment by JasperVanDenBosch** [comment]
version 0.0.1

**Comment by leehweeling** [comment]
Hi,

I'm also experiencing similar issues. Has anyone the solution to this?

Cheers,
Hweeling

**Comment by JasperVanDenBosch** [comment]
Yes this will be resolved in the next few days. I'll post an update here

**Comment by JasperVanDenBosch** [comment]
@tbennett4 @leehweeling please update `rsatoolbox` to version `0.0.2` and try again. You should be able to update with:

```sh
pip install -U rsatoolbox
```

**Comment by JasperVanDenBosch** [comment]
Closing now as this seems to be resolved in #193. Feel free to re-open or start a new issue if this re-occurs in version 0.0.2
